content/dart/concepts/queue/terms/add/add.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: '.add()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'A Dart method to append or concatenate two strings.' 
+Subjects: 
+  - 'Developer Tools'
+  - 'Code Foundations'
+Tags:
+  - 'Dart Operators'
+  - 'Queues'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-dart'
+  - 'paths/dart'
+---
+
+Dart allows the user to append or concatenate a string and add it to the end of an existing string using the **.add()** method. Since Dart strings are immutable (unable to be changed once created), the .add() method creates a new string rather than modifying the original string.
```

Comment:
```suggestion
Dart allows the user to append or concatenate a string and add it to the end of an existing string using the **`.add()`** method. Since Dart strings are immutable (unable to be changed once created), the `.add()` method creates a new string rather than modifying the original string.
```